packages/genesys-spark-components/src/components/beta/gux-side-sheet/gux-side-sheet.tsx

```diff
@@ -0,0 +1,40 @@
+import { Component, h, Element, Prop } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxSideSheetSize } from './gux-side-sheet.types';
+
+@Component({
+  tag: 'gux-side-sheet-beta',
+  styleUrl: 'gux-side-sheet.scss',
+  shadow: true
+})
+export class GuxSideSheet {
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  size: GuxSideSheetSize = 'small';
+
+  componentWillLoad(): void {
+    trackComponent(this.root);
+  }
+
+  render(): JSX.Element {
+    return (
+      <div class={`gux-side-sheet gux-side-sheet-${this.size}`}>
+        <header>
+          <div class="gux-side-sheet-icon">
+            <slot name="icon" />
+          </div>
+          <slot name="heading" />
+          <gux-dismiss-button />
```

Comment:
I dont think the `gux-dismiss-button` is currently positioned correctly. I think it should be positioned `absolute` with a `top` and `right` of `12px`